content/blog/introducing-kx/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Introducing kx: Kubernetes for Everyone"
+authors: ["levi-blackstone"]
+tags: ["Kubernetes"]
+meta_desc: "The Kubernetes Extensions (kx) library for Pulumi is designed to simplify the declaration of Kubernetes resources, and make the API easier to use."
+date: "2019-11-14"
+
+meta_image: "feature.png"
+---
+
+Using the Kubernetes API to deploy and manage workloads often feels heavy and
+repetitive today. Many of the API fields are deeply nested and require users
+to specify the same values redundantly across different resources. While this
+explicit specification is necessary for Kubernetes to operate, it’s not friendly
+to the people writing it.
```

Comment:
not always friendly?  (something to make this a little softer and more inviting).

Another point you could make it that this leads to lots of copy/pasting and boilerplate.